style-guide/src/components/design-nav/design-nav.js

```diff
@@ -13,15 +20,15 @@ const NavLink = (props) => (
 export default () => (
   <div className={styles.navSection}>
     <h4>DESIGN TOKENS</h4>
-    <NavLink exact to="/">All Design Tokens</NavLink>
-    <NavLink to="/otkit/colors">Colors</NavLink>
-    <NavLink to="/otkit/typography">Typography</NavLink>
-    <NavLink to="/otkit/grids">Grids</NavLink>
-    <NavLink to="/otkit/spacing">Spacing</NavLink>
-    <NavLink to="/otkit/borders">Borders</NavLink>
-    <NavLink to="/otkit/breakpoints">Breakpoints</NavLink>
-    <NavLink to="/otkit/shadows">Shadows</NavLink>
-    <NavLink to="/otkit/icons">Icons</NavLink>
-    <NavLink to="/otkit/icons-theme">Icons (theme)</NavLink>
+    <NavLink to="/">All Design Tokens</NavLink>
+    <NavLink to="/otkit/colors/">Colors</NavLink>
+    <NavLink to="/otkit/typography/">Typography</NavLink>
+    <NavLink to="/otkit/grids/">Grids</NavLink>
+    <NavLink to="/otkit/spacing/">Spacing</NavLink>
+    <NavLink to="/otkit/borders/">Borders</NavLink>
+    <NavLink to="/otkit/breakpoints/">Breakpoints</NavLink>
+    <NavLink to="/otkit/shadows/">Shadows</NavLink>
+    <NavLink to="/otkit/icons/">Icons</NavLink>
+    <NavLink to="/otkit/icons-theme/">Icons (theme)</NavLink>
```

Comment:
If there is no trailing slashes and if we refresh the page, the slash will automatically append to it and we won't be able to match the path and the styling will not apply.

We can use this if we want to keep non-trailing slash https://www.gatsbyjs.com/plugins/gatsby-plugin-remove-trailing-slashes/